modules/cnf-performing-end-to-end-tests-for-platform-verification.adoc

```diff
@@ -24,6 +24,8 @@ Validations include:
 * Having the PTP Operator installed
 * Using OVN kubernetes as the SDN
 
+Latency tests, a part of the CNF-test container, also require the same validations. For more information about running a latency test, see link:cnf-performing-end-to-end-tests-running-the-latency_tests_{context}[Running the latency tests].
```

Comment:
Unfortunately, we can't include any internal links (xrefs) in modules right now. We're hoping that xrefs can be reintroduced soon with Pantheon 2. So, you have to either have an unlinked reference to that section or remove the reference altogether. :disappointed: